cttportfolio.html

```diff
@@ -0,0 +1,173 @@
+									</a>
+								</div>
+							</div>
+						</div>
+					</div>
+					<div class="project_container project2">
+						<div class="project_text">
+							<h5>main project 2 name here</h5>
+							<p>Project 2 text here links below this somewhere too</p>
+							<div class="project_links project_main_links">
+								<div class="project_icon_size">
+									<a class="icon_hover">
+										<img src="https://cdn2.iconfinder.com/data/icons/black-white-social-media/64/social_media_logo_github-512.png" alt="github icon here">
+									</a>
+								</div>
+								<div class="project_icon_size">
+									<a class="icon_hover" href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">
+										<img src="https://cdn2.iconfinder.com/data/icons/black-white-social-media/32/youtube_online_social_media_tube-512.png" alt="youtube icon here">
+									</a>
+								</div>
+								<div class="project_icon_size">
+									<a class="icon_hover">
+										<img src="https://cdn2.iconfinder.com/data/icons/black-white-social-media/32/twitter_online_social_media-512.png" alt="twitter icon here">
+									</a>
+								</div>
+							</div>
+						</div>
+							<a class="pf_link icon_hover" href="cttprojectpage.html">
+						<img class="project_img" src="https://via.placeholder.com/640x360" alt="project picture here">
+						</a>
+					</div>
+				</div>
+			</div>
+			<div class="pf_sub">
+				<a class="pf_sub_links icon_hover" href="cttprojectpage.html">
+					<div class="pf_sub_link_div">
+						<h6>sub project name 1</h6>
+						<img src="https://www.webfx.com/blog/images/cdn.designinstruct.com/files/582-how-to-image-placeholders/generic-image-placeholder.png" alt="sub project pic here">
+					</div>
+				</a>
+				<a class="pf_sub_links icon_hover" href="cttprojectpage.html">
+					<div class="pf_sub_link_div">
+						<h6>sub project name 2</h6>
+						<img src="https://www.webfx.com/blog/images/cdn.designinstruct.com/files/582-how-to-image-placeholders/generic-image-placeholder.png" alt="sub project pic here">
+					</div>
+				</a>
+				<a class="pf_sub_links icon_hover" href="cttprojectpage.html">
+					<div class="pf_sub_link_div">
+						<h6>sub project name 3</h6>
+						<img src="https://www.webfx.com/blog/images/cdn.designinstruct.com/files/582-how-to-image-placeholders/generic-image-placeholder.png" alt="sub project pic here">
+					</div>
+				</a>
+			</div>
+		</div>
+		<div id="contactme_jump" class="contactme">
+			<h4>Contact me</h4>
+			<div class="contactmetext">
+				<div class="contact_div background_color">
+					<p>Want to get in touch?</p>
+					<h6>E-mail: christianturner@hotmail.co.uk</h6>
+				</div>
+				<div class="contact_input background_color">
+					<input placeholder="Enter contact information" type="email">
+					<button class="icon_hover">Send</button>
+				</div>
+			</div>
+		</div>
+	</main>
+	<footer>
+		<div class="footer_links">
```

Comment:
do you need a div inside the footer? can you not put the styles on the footer tag and not have the div?